libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/RoadObjectDistanceInfo.kt

```diff
@@ -0,0 +1,45 @@
+package com.mapbox.navigation.base.trip.model.roadobject
+
+/**
+ * RoadObjectDistanceInfo contains information about distance to the road object of a concrete
+ * flavor/shape (gantry, polygon, line, point etc.)
+ *
+ * @param distanceInfoType type of the distance info object
+ */
+abstract class RoadObjectDistanceInfo(
+    @RoadObjectDistanceInfoType.Type val distanceInfoType: Int,
+) {
+
+    /**
+     * Distance to start of the object from current location
+     */
+    abstract val distanceToStart: Double
```

Comment:
It seems we're not included this as part of `equals`, `hashCode` and `toString`. Is that expected? 🤔

Reply:
fixed